public/catalog/v1.0.0/openapi.json

```diff
@@ -1454,6 +1454,12 @@
             "type": "string",
             "title": "External url",
             "description": "The external url of the service instance used with relation to this order item"
+          },
+          "insights_request_id":{
+            "type": "string",
+            "title": "Insights Request ID ",
+            "example": "insights-request-id-string",
```

Comment:
@lindgrenj6 Can we add a description that states
"The insights request id can be used to collect log data for this order item as its processed by the system"